libminifi/include/Site2SiteClientProtocol.h

```diff
@@ -387,6 +387,15 @@ class DataPacket {
 
 };
 
+/**
+  * Site2Site Peer
+  */
+ typedef struct Site2SitePeerStatus {
+   std::string host_;
+   int port_;
+   bool isSecure_;
+ } Site2SitePeerStatus;
```

Comment:
What about flowFileCount, the number of flow files the peer holds? That is discoverable using the `/site-to-site/peers` endpoint and could be useful if the client wants to use it for load balancing decisions.